pkg/devices/led.go

```diff
@@ -71,16 +83,36 @@ func ledWrite(_ *sdk.Device, data *sdk.WriteData) error {
 		if len(decoded) != 3 {
 			return fmt.Errorf("color value should be a 3-byte (RGB) hex string")
 		}
-		color = string(raw)
+
+		dState, ok := deviceState[device.ID()]
+		if !ok {
+			deviceState[device.ID()] = map[string]interface{}{color: decoded}
```

Comment:
@lazypower  derp.. I missed this before, so my bad..

`decoded` here is the hex string decoded into a `[]int` (to validate that it is indeed hex), but the `Read` function assumes its a string, e.g. `color = dState[color].(string)`, so it will fail the type conversion there. 

easiest thing to do is instead of storing `decoded` here, store `string(raw)`

Reply:
doh, i'll include this in the lock pr over in #43